http_logs/track.py

```diff
@@ -1,9 +1,9 @@
 from copy import copy
 import re
 
-from esrally import exceptions
```

Comment:
See comment below, if we go the backwards compatibility route, we need to keep it imported.